data/iupac_goldbook/transform.py

```diff
@@ -0,0 +1,140 @@
+    data = requests.get(data_path)
+    with open(fn_data_original, "wb") as f:
+        f.write(data.content)
+    del data
+
+    # create dataframe
+    with open(fn_data_original) as f:
+        data = json.load(f)
+
+    entries = []
+    for key in data["entries"].keys():
+        d = data["entries"][key].copy()
+        d["entry_id"] = int(key)
+        entries.append(d)
+    df = pd.DataFrame(entries)
+
+    # check if fields are the same
+    fields_orig = df.columns.tolist()
+    assert fields_orig == [
+        "code",
+        "term",
+        "index",
+        "status",
+        "doi",
+        "version",
+        "lastupdated",
+        "newcode",
+        "url",
+        "definition",
+        "identifiers",
+        "related",
+        "entry_id",
+        "replacedby",
+    ]
+
+    # save to csv
+    fn_data_csv = "data_clean.csv"
+    df.to_csv(fn_data_csv, index=False)
+
+    # create meta yaml
+    meta = {
+        "name": data[
+            "title"
+        ],  # unique identifier, we will also use this for directory names
+        "description": """The Compendium is popularly referred to as the Gold
+Book, in recognition of the contribution of the late Victor Gold, who
+initiated work on the first edition. It is one of the series of IUPAC
+Colour Books on chemical nomenclature, terminology, symbols and units
+(see the list of source documents), and collects together terminology
+definitions from IUPAC recommendations already published in Pure and
+Applied Chemistry and in the other Colour Books. Terminology
+definitions published by IUPAC are drafted by international committees
+of experts in the appropriate chemistry sub-disciplines, and ratified
+by IUPAC's Interdivisional Committee on Terminology, Nomenclature and
+Symbols (ICTNS). In this edition of the Compendium these IUPAC-approved
+definitions are supplemented with some definitions from ISO and from
+the International Vocabulary of Basic and General Terms in Metrology,
+both these sources are recognised by IUPAC as authoritative. The result
+is a collection of nearly 7000 terms, with authoritative definitions,
+spanning the whole range of chemistry.""",
+        "targets": [
+            {
+                "id": "definition",  # name of the column in a tabular dataset
+                "description": "definition of a chemistry term",  # description of what this column means
+                "units": None,  # units of the values in this column (leave empty if unitless)
+                "type": "text",  # can be "categorical", "ordinal", "continuous", "text"
+                "names": [  # names for the property (to sample from for building the prompts)
+                    "definition",
+                    "definition of a chemistry term",
+                ],
+            },
+        ],
+        "identifiers": [
+            {
+                "id": "term",  # column name
+                "type": "OTHER",  # can be "SMILES", "SELFIES", "IUPAC", "OTHER"
```

Comment:
```suggestion
                "type": "Other",  # can be "SMILES", "SELFIES", "IUPAC", "OTHER"
```

Reply:
Good catch, I just used `OTHER`  because we also have it with capital letters in the comment
`"type": "Other",  # can be "SMILES", "SELFIES", "IUPAC", "OTHER"`